src/docker_entrypoint.py

```diff
@@ -11,7 +11,7 @@
 import sys
 import os
 import pwd
-import subprocess  # nosec
+import subprocess
 
 HOST_UID = os.stat("/app").st_uid
 HOST_USER = "james"
```

Comment:
?

Reply:
It is important that the UUID maps correctly (that's what makes this script work) but the username doesn't matter. The original script that I lifted from [django-template](https://github.com/18F/django-template/blob/main/django_template/templates/docker_entrypoint.py.jinja) used `HOST_USER = env.get_credential("HOST_USER", "{{ app_name }}_user")`